src/util/LoadingButton.tsx

```diff
@@ -10,17 +11,36 @@ interface Props {
   type?: ButtonProps['type'];
   disabled?: boolean;
   onClick?: onClick;
+  toolTipText?: string;
```

Comment:
Refactor rename this to `disabledToolTipText`
```suggestion
  disabledToolTipText?: string;
```